melodic/distribution.yaml

```diff
@@ -3806,6 +3806,26 @@ repositories:
       url: https://github.com/ros-gbp/libg2o-release.git
       version: 2018.3.25-0
     status: maintained
+  librealsense:
+    doc:
+      type: git
+      url: https://github.com/IntelRealSense/librealsense.git
+      version: legacy
+    source:
+      type: git
+      url: https://github.com/IntelRealSense/librealsense.git
+      version: legacy
+    status: maintained
+  librealsense2:
+    doc:
+      type: git
+      url: https://github.com/IntelRealSense/realsense-ros.git
+      version: development
+    source:
+      type: git
+      url: https://github.com/IntelRealSense/realsense-ros.git
```

Comment:
This is confusing, I'm not sure you are doing what you intend to here.  What you are calling `librealsense2` here is being linked to realsense-ros which contains two ros packages and one of them depends on librealsense2 which this package doesn't supply.  Instead, what you are calling `librealsense` above is the git repo that supplies the `librealsense2` package.